war/src/main/js/upgradeWizard.js

```diff
@@ -1,40 +1,38 @@
-onSetupWizardInitialized(function(wizard) {
```

Comment:
@kzantow What do you think about [this](https://github.com/jenkinsci/jenkins/pull/2368/commits/f743ef9311523ca2ae0cd82823a1082cb8e62119?w=1)?